### ISSUE TITLE
Reduce the size of blurry placeholder dataURLs

### ISSUE THREAD

**Issue by atcastle** [vercel/next.js#26146]
### Describe the feature you'd like to request

Currently the dataURLs used to generate the blurry placeholders are fairly large when inlined into the initial page document. Images formatted as jpeg are noticeably larger than images formatted as png.

Especially if a user includes a lot of blurry placeholders on a page, this could cause a worsening of page load metrics as the large number of inlined dataURLs inflates the size of the initial document.

### Describe the solution you'd like

A first step to improving placeholder performance will be to generate smaller inline dataURLs. Since we can see that at the placeholder resolution (~8 pixels wide) jpegs are a lot larger than pngs, we should probably generate all placeholders in png format.

Questions I have here are:
1) Is it faster to transcode images from jpeg > png then resize, or to resize first then transcode?
2) What is the estimated effect of doing this transcoding on build time?
3) Is png actually the smallest format for a tiny image? We should experiment with other formats.

### Describe alternatives you've considered

We could also look into Blurhash again, although I'd prefer a solution that doesn't require any client-side JS to decode.

**Comment by matamatanot** [comment]
I'm not sure if `Blurhash` should be the default strategy, but it would be nice if it could be supported.
I've been using Next.js since before 10, and at the same time I'm using `Blurhash` and `react-blurhash`. I'm using next/image now, but I still can't get rid of `react-blurhash` because the server side returns the blurhash string.

**Comment by styfle** [comment]
> We could also look into Blurhash again, although I'd prefer a solution that doesn't require any client-side JS to decode.

Agreed. If we implement support for https://blurha.sh strings, then we would want to convert it to a Data URL string on the server-side so the initial render can immediately load the blur-up placeholder. I think this conversion could be achieved with something like [`@napi-rs/image`](https://www.npmjs.com/package/@napi-rs/image).

**Comment by lachlanjc** [comment]
A new library for this was released today: https://evanw.github.io/thumbhash/
